src/code/float-trap.lisp

```diff
@@ -117,6 +117,8 @@ in effect."
       (setf (ldb float-precision-control modes)
             (or (cdr (assoc precision +precision-mode-alist+))
                 (error "unknown precision mode: ~S" precision))))
+    ;; FIXME: This apparently doesn't work on Darwin
```

Comment:
is this something that you think you can address in the long-run?  Or a copy of an old comment?

Reply:
This is a copy from old comment. I'd like to keep it that way.

Reply:
@catap @csrhodes Since I have some interest (and relevance) in rounding on PPC, I could look into that in a while, but I will need someone’s assistance with Lisp code here.

Reply:
@barracuda156 feel free to ask directly :)